includes/external-connection-cpt.php

```diff
@@ -654,13 +681,12 @@ function add_menu_item() {
 		 * Filter Distributor capabilities allowed to view external connections.
 		 *
 		 * @since 1.0.0
-		 * @hook dt_capabilities
 		 *
 		 * @param {string} 'manage_options' The capability allowed to view external connections.
 		 *
 		 * @return {string} The capability allowed to view external connections.
 		 */
-		apply_filters( 'dt_capabilities', 'manage_options' ),
+		 'read',
```

Comment:
I think we're fine to remove this filter but we should also remove the filter docblock above this, as that's no longer needed